src/pages/Timeline/AddTimelineEntry/AddTimelineEntry.module.css

```
@@ -0,0 +1,134 @@
+* {
+    box-sizing: border-box;
+}
+
+.container {
+    --darkgrey: #666;
+    width: min(80%, 1000px);
+    margin: 0 auto;
+}
+
+.header {
+    text-align: start;
+    font-family: 'Source Sans Pro';
+    padding-top: 100px;
+    font-size: 40px;
+    font-weight: 600;
+}
+
+.subHeader {
+    font-family: 'Source Sans Pro';
+    font-size: 18px;
+    font-weight: 500;
+    margin: 0;
+    margin-top: 8px;
+}
+
+main {
+    margin: 36px 0;
+}
+
+.gridContainer {
+    display: grid;
+    grid-template-columns: 3fr 2fr 5fr;
+    gap: 28px;
+}
+
+.basicInfo {
+    display: flex;
+    flex-direction: column;
+    gap: 28px;
+}
+
+.descriptionGroup {
+    text-align: start;
+    grid-column: 3 / 4;
+    align-self: stretch;
+}
+
+.descriptionGroup label {
+    display: block;
+    font-family: 'Source Sans Pro';
+    font-size: 16px;
+    color: #777;
+    font-weight: 600;
+    margin-bottom: 8px;
+}
+
+.descriptionGroup textarea {
+    width: 100%;
+    height: calc(100% - 30px);
+    border: 1px solid #ddd;
+    border-radius: 2px;
+    padding: 8px;
+    font-family: 'Source Sans Pro';
+    font-size: 16px;
+    color: #333;
+    resize: none;
+}
+
+textarea:focus {
+    outline-color: black;
+}
+
+.controls {
+    width: 100%;
+    margin-top: 24px;
+    display: flex;
+    justify-content: flex-end;
+    gap: 24px;
+}
+
+.cancelButton {
+    background-color: white;
+    color: #555;
+    border: 1px solid transparent;
+    padding: 4px 16px;
+    border-radius: 2px;
+    cursor: pointer;
+    font-family: 'Source Sans Pro';
+    font-size: 20px;
+}
+
+.saveButton {
```

I would suggest changing this name to the following:

```suggestion
.addButton {
```
**or** 

```suggestion
.add-button {
```

---

oh right! nice catch, I didn't realize